packages/caldav/README.md

```diff
@@ -0,0 +1,33 @@
+# timeld - CalDAV integration
+
+This packages supports live update of timesheet entries:
+- _from_ a CalDAV end-point
```

Comment:
I'd suggest adding an example here along the lines of "such as iCloud Calendar".